app/bingey-ui/src/admin/components/Navigation/AppBar.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import { Tab, Tabs } from '@material-ui/core';
+import { withRouter } from 'react-router-dom';
+import './AppBar.scss';
+
+const Header = withRouter(({ history }) => {
+  const [value, setValue] = useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+  return (
+    <div>
```

Comment:
This `div` doesn't seem to serve any purpose as this only returns one base component